source/css/sass/_normalize.scss

```diff
@@ -51,47 +51,64 @@ h6 {
   @include h6-spacing();
 }
 
-p {
-  @include body-para();
+address,
+article,
```

Comment:
Probably shouldn't include sectioning content here?

Reply:
Had to add https://github.com/libero/pattern-library/blob/8b1b20a399912c6963db047586ec85f5ada39714/source/css/sass/mixins/_grid.scss#L7 as it is currently included.